sdk/storage/Azure.Storage.Blobs/tests/EncryptedBlockBlobClientTests.cs

```diff
@@ -18,25 +25,281 @@ public EncryptedBlockBlobClientTests(bool async)
         {
         }
 
-        // Placeholder test to verify things work end-to-end
-        [Test]
-        public async Task DeleteAsync()
+        #region Utility
+
+        /// <summary>
+        /// Gets a client to a nonexistent blob using client-side encryption in a brand new disposable container.
+        /// </summary>
+        /// <param name="blob">The blob client created.</param>
+        /// <returns>The IDisposable to delete the container when finished.</returns>
+        private IDisposable GetEncryptedBlockBlobClient(out EncryptedBlobClient blob, MockKeyEncryptionKey mock)
+        {
+            var disposable = this.GetNewContainer(out var container);
+            blob = new EncryptedBlobClient(
+                    new Uri(Path.Combine(container.Uri.ToString(), this.GetNewBlobName())), this.GetNewSharedKeyCredentials(),
+                    keyEncryptionKey: mock,
+                    keyResolver: mock);
```

Comment:
You also need our standard test BlobClientOptions passed in: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs#L46